pgmoon/json.moon

```diff
@@ -0,0 +1,10 @@
+encode_json = (tbl) ->
+  json = require "cjson"
+  enc = json.encode tbl
+  "'#{(enc\gsub "'", "''")}'"
```

Comment:
I just updated the patch to use the same `escape_literal`, and to receive a custom escaping function as the array module.